### PR TITLE
chore: update the 'CODEOWNERS' file to include directories relevant to the 'sdk-devs' team

### PR DESCRIPTION
**Pull request by omridan159**
## **Description**

This PR updates `CODEOWNERS` file to include directories relevant to the @MetaMask/sdk-devs team

## **Related issues**

Fixes: NA

## **Manual testing steps**

NA

### **Before**

NA

### **After**
NA

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by abretonc7s — DISMISSED** [review]

**Review by abretonc7s — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,3 +9,9 @@ app/components/UI/Ramp/              @MetaMask/ramp @MetaMask/mobile-platform
 app/reducers/fiatOrders/             @MetaMask/ramp @MetaMask/mobile-platform
 app/components/Views/confirmations   @MetaMask/confirmations-ux @MetaMask/mobile-platform
 ppom                                 @MetaMask/confirmations-ux @MetaMask/mobile-platform
+app/core/DeeplinkManager   @MetaMask/sdk-devs
```
Comment (sethkfman) [inline .github/CODEOWNERS:12]: I would like to start commenting the CODEOWNER file. Can you Add a comment above this line describing the domain of these files? Similar to what you see [here](https://github.com/MetaMask/metamask-extension/blob/be3c4f277d13ff038bb551bcfc4149642a003ee1/.github/CODEOWNERS#L55)

Comment (sethkfman) [inline .github/CODEOWNERS:12]: Can you also add the `@MetaMask/mobile-platform` to each of these? The idea here is that the platform team should be able to approve changes/merges for releases only. We will reach out to your team if there are changes specific to your domain. Thx

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by andreahaku — DISMISSED** [review]
LGTM

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,3 +9,14 @@ app/components/UI/Ramp/              @MetaMask/ramp @MetaMask/mobile-platform
 app/reducers/fiatOrders/             @MetaMask/ramp @MetaMask/mobile-platform
 app/components/Views/confirmations   @MetaMask/confirmations-ux @MetaMask/mobile-platform
 ppom                                 @MetaMask/confirmations-ux @MetaMask/mobile-platform
+app/actions/sdk   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Approvals/WalletConnectApproval   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Views/SDK   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Views/WalletConnectSessions   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/core/BackgroundBridge/WalletConnectPort.ts   @MetaMask/sd
```
Comment (sethkfman) [inline .github/CODEOWNERS:16]: @omridan159 You have a typo here
```suggestion
app/core/BackgroundBridge/WalletConnectPort.ts   @MetaMask/sdk-devs @MetaMask/mobile-platform
```

**Review by sethkfman — DISMISSED** [review]
LGTM

**Review by NicolasMassart — DISMISSED** [review]
Please add description of the changes outside .github/CODEOWNERS in the PR description. The current description makes think that only .github/CODEOWNERS should be modified in this PR and it requires extra effort to figure out that you also reorganised files for your need and not only updated the CODEOWNERS.

- creation of app/components/Views/SDK/SDKSessionsManager/getSharedStyles.ts
- moving files

Why is it important?
- modifying only the .github/CODEOWNERS has only an impact on Github and the PR process.
- moving files and extracting styles are changes on the production code and can actualy generate bugs.
The risk level for thes two kind of changes is very different and should be at least clearly described in the PR. Ideally it should even be 2 distinct PRs.

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by andreahaku — DISMISSED** [review]
LGTM

**Review by sethkfman — DISMISSED** [review]
LGTM

**Comment by christopherferreira9** [comment]
Hi @sethkfman ! SDK specific tests will be performed after merging this PR in order to have multiple PR's tested at the same time given these do not contain changes other than impor changes. 
We're bumping into Sonarcloud failures probably due to this https://github.com/MetaMask/metamask-mobile/pull/9435. Can we bypass this since we're adding the unit tests on a separate PR?

**Review by NicolasMassart — DISMISSED** [review]

**Review by Cal-L — APPROVED** [review]
LGTM. E2E passed in prior commits

**Pull request merged**